src/scenarios/shared/precommands.py

```diff
@@ -49,6 +49,10 @@ def __init__(self):
 
         args = parser.parse_args()
 
+        if not args.operation:
+            getLogger().error("Please specify an operation: %s" % list(OPERATIONS))
+            sys.exit(1)
```

Comment:
Why this instead of the previous just requiring an operation?

Reply:
```required``` parameter in ArgumentParser.add_subparser is only supported beginning in python3.7.  Doing this workaround because currently [we have trouble updating the python version in VMs](https://github.com/dotnet/core-eng/issues/9647).  Before the python version thing get fixed I think it's good to just avoid python3.7 completely.

Reply:
Ah, got it. :(